packages/instantsearch.css/CHANGELOG.md

```diff
@@ -1 +1,14 @@
+# Change Log
+
+All notable changes to this project will be documented in this file.
+See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
+
+## [7.4.6](https://github.com/algolia/instantsearch.js/compare/instantsearch.css@7.4.5...instantsearch.css@7.4.6) (2022-12-27)
+
+**Note:** Version bump only for package instantsearch.css
```

Comment:
annoying these kind of bumps can't be skipped. Maybe it can be checked after the version script and reverted? 

Maybe first listing which packages have changes, then only version those?